.circleci/config.yml

```diff
@@ -159,14 +173,25 @@ workflows:
           matrix:
             parameters:
               version:
-                - "3.12"
+                - "3.9"
```

Comment:
Should this be 3.12 or is there a specific reason to use 3.9 here and below

Reply:
Ive set all python versions to 3.12 if not tested for all